.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = ../../euclideon/udcore.git
 [submodule "3rdParty/imgui"]
 	path = 3rdParty/imgui
-	url = ../../ocornut/imgui.git
+	url = ../../ocornut/imgui.git
```

Comment:
Probably should have just reverted this file, you've removed the trailing empty line.